tests/rptest/tests/scaling_up_test.py

```diff
@@ -475,14 +475,15 @@ def verify_disk_usage(usage: dict, added_ids: list, percentage: float):
             for id in added_ids:
                 added_node_usage = usage[id]
                 assert added_node_usage < percentage * avg_usage, \
-                f"Added node {id} disk usage {added_node_usage} is too large, "
-                f"expected usage to be smaller than {percentage * avg_usage} bytes"
+                f"Added node {id} disk usage {added_node_usage} is too large, \
+                  expected usage to be smaller than {percentage * avg_usage} bytes"
```

Comment:
won't the whitespace in the beginning of this line be part of the format string this way?
```
>>> a=1
>>> print(f"a=\
...                                 {a}")
a=                                1
>>>
```